src/core-asyncpipeline/AsyncPipeline/AsyncPipeline.T/AsyncPipeline.T.cs

```diff
@@ -0,0 +1,27 @@
+#nullable enable
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System
+{
+    public readonly partial struct AsyncPipeline<T> : IEquatable<AsyncPipeline<T>>
+    {
+        private static readonly CancellationToken CanceledToken;
+
+        static AsyncPipeline() => CanceledToken = new(canceled: true);
+
+        private readonly ValueTask<T> valueTask;
+
+        private readonly bool hasCanceled;
```

Comment:
Just in case - is it ok that the default pipeline is not in cancellation state?
Should the default value be treated as "Start"?

Reply:
I think it's okey. CancellationToken and ValueTask are also not cancelled by default

Reply:
Ok